code/modules/reagents/chemistry/reagents/food_reagents.dm

```diff
@@ -243,9 +243,18 @@
 /datum/reagent/consumable/sodiumchloride
 	name = "Table Salt"
 	id = "sodiumchloride"
-	description = "A salt made of sodium chloride. Commonly used to season food."
+	description = "Sodium, atomic number 11, was first isolated by Peter Dager in 1807. A chemical component of salt, he named it in honor of the saltiest station in space, [Current station name]."
```

Comment:
You can't have functions execute before runtime like this, it'd have to go in `New()`
You also use `station_name()` to return the current world name.